build_runner_core/lib/src/asset_graph/node.dart

```diff
@@ -30,25 +30,30 @@ class AssetNode {
 
   /// The assets that any [Builder] in the build graph declares it may output
   /// when run on this asset.
-  final Set<AssetId> primaryOutputs = <AssetId>{};
+  final Set<AssetId> _primaryOutputs;
+  Iterable<AssetId> get primaryOutputs => _primaryOutputs;
```

Comment:
If you are supposed to go via the `inspect` below, why have these?

Reply:
You only need to go via `inspect` if the method you want is on `Set`; almost all are already on `Iterable`.